test/node/msw-api/req/passthrough.node.test.ts

```diff
@@ -9,6 +9,15 @@ const httpServer = new HttpServer((app) => {
   app.post<never, ResponseBody>('/user', (req, res) => {
     res.json({ name: 'John' })
   })
+  app.post('/code/204', (req, res) => {
```

Comment:
Nitpick: why not

```js
app.post('/code/:code', (req, res) => {
  res.status(req.params.code).end()
})
```

Reply:
no reason, really.  I went back and forth on this actually, and settled on the explicit handlers, since it's a but easier to parse quickly that we're only defining a couple codes that we care about vs arbitrary ones (especially if we typod a non-numeric code)